roles/icinga_plugins/molecule/default/prepare.yml

```diff
@@ -8,18 +8,37 @@
         state: installed
       when: ansible_os_family == 'RedHat'
 
+    - name: install git on Redhat based systems
```

Comment:
you could use something like this to only have one task.
```
- name: install git on Redhat based systems
  ansible.builtin.package:
    name: git
    state: present
    update_cache: "{{ (ansible_pkg_mgr == 'apt') | ternary('yes', omit) }}"
    cache_valid_time: "{{ (ansible_pkg_mgr == 'apt') | ternary(cache_timeout, omit) }}"
```

Reply:
changed in https://github.com/T-Systems-MMS/ansible-collection-icinga/pull/34/commits/84468082d660f88db31c14cf419297ca253ccb5a